### PR TITLE
split

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-11-30-split-keyboard.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+categories: [Writing,Programming]
+image:
+image2:
+author: Valentino Urbano
+---
+
+
+# Why a Split Keyboard?
+
+I've been reading a lot about [carpal tunnel syndrome][1] in the last few days and I'm trying to be mentally aware of the position of my hands while typing. it seems that on a normal keyboard you cannot keep it aligned since the keys are too close to one another and either the right or the left wrist will have to tilt to reach certain letters. The solution would be to try a split keyboard or a keyboard that has some space between the right and left side of the keyboard. I know one person in the office is already using it and from what he tells me he can now type faster than on a normal keyboard.
```
Comment (MsSlacker) [inline _posts/2019-11-30-split-keyboard.md:17]: Capitalise "it" in 2nd sentence

Comment (MsSlacker) [reply _posts/2019-11-30-split-keyboard.md:17]: Is - > that is?

File: _posts/2019-11-30-split-keyboard.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Having the keyboard split and aligned to the resting position of your wrists means that you do not have to twist your wrist much at all.
```
Comment (MsSlacker) [inline _posts/2019-11-30-split-keyboard.md:19]: Seems a bit short... And doesn't really add anything to what you already wrote on the subject before?

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-11-30-split-keyboard.md (hunk 2 above)
Comment (valeIT) [reply _posts/2019-11-30-split-keyboard.md:19]: allright, will just link the article from the other one and delete this one

Comment (MsSlacker) [reply _posts/2019-11-30-split-keyboard.md:19]: Can still put some of this on the other on I guess... No need to delete...

**Comment by valeIT** [comment]
Closing, merged into #94

**Pull request closed without merging**